packages/cli/src/commands/buildHandler.js

```diff
@@ -7,14 +8,14 @@ import { rimraf } from 'rimraf'
 import terminalLink from 'terminal-link'
 
 import { recordTelemetryAttributes } from '@redwoodjs/cli-helpers'
-import { buildApi, cleanApiBuild } from '@redwoodjs/internal/dist/build/api'
-import { generate } from '@redwoodjs/internal/dist/generate/generate'
-import { loadAndValidateSdls } from '@redwoodjs/internal/dist/validateSchema'
-import { detectPrerenderRoutes } from '@redwoodjs/prerender/detection'
+import { buildApi, cleanApiBuild } from '@redwoodjs/internal/dist/build/api.js'
+import { generate } from '@redwoodjs/internal/dist/generate/generate.js'
+import { loadAndValidateSdls } from '@redwoodjs/internal/dist/validateSchema.js'
+import { detectPrerenderRoutes } from '@redwoodjs/prerender/detection/index.js'
 import { timedTelemetry } from '@redwoodjs/telemetry'
 
-import { getPaths, getConfig } from '../lib'
```

Comment:
This is ESM-specific. Implicit imports like this (this is really `'../lib/index.js'`) don't work